R/input_output_filter.R

```diff
@@ -289,6 +313,12 @@ match_datasets = function(ds1, ds2){
 #' @param out_fp The output filepath.
 #' @param map_fp (OPTIONAL) The metadata map output filepath if you want to 
 #'  write it to file.
+#' @examples 
+#' \dontrun{
+#' export_otu_table(input, 'desired_file_path', 
```

Comment:
this is a good place to include the \dontrun because we don't want files to actually get created when running the examples